_includes/smartstate.md

```diff
@@ -1,16 +1,13 @@
 After enabling the required server roles, enable SmartState analysis.
-See [Smart State Analysis
-Support](https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html-single/support_matrix/#smart_state_analysis_support)
-from the Support Matrix and [Running a SmartState
-Analysis](https://access.redhat.com/documentation/en-us/red_hat_cloudforms/4.7/html-single/managing_providers/#running-a-smartstate-analysis)
-in the *Managing Providers* guide for more information.
+See [Smart State Analysis Support](https://www.ibm.com/support/knowledgecenter/SSFC4F_2.0.0/mcm/infrastructure/support_matrix.html#smart-state-analysis-support)
+from the Support Matrix.
```

Comment:
@Fryguy wondering if this should be pointing to the upstream [Capabilities Matrix](https://www.manageiq.org/docs/reference/latest/capabilities_matrix/index.html#smart-state-analysis-capabilities)

Reply:
@chessbyte Should we replace the downstream "support matrix" with the upstream "capabilities matrix"?  If so, I can make these changes and point to the capabilities matrix in the smartstate.md so both upstream and downstream are in sync.

Reply:
@chessbyte In the downstream "support matrix" is where we have documented the "tiered support" (e.g. https://www-03preprod.ibm.com/support/knowledgecenter/SSFC4F_2.0.0/mcm/infrastructure/support_matrix.html#supported-providers) and I don't see this in the upstream capabilities matrix.  The Support Matrix was a Red Hat CloudForms only doc when we initially published these downstream docs.  Once you created the capabilities matrix doc from the support matrix doc, some items we need downstream are not included I think. If we point to the capabilities matrix upstream then that would need to be available downstream as well but it's lacking some content from the support matrix.

Reply:
because for an upstream product, there is only community support (i.e. best-effort basis), which is why I renamed it from Support to Capabilities.  In the upstream docs, we really should not have links to downstream, so it can stand on its own.

Reply:
Oh thanks...I missed that link in review